bodhi/server/templates/update.html

```diff
@@ -1045,7 +1045,7 @@ <h3>Related Bugs <span class="badge">${len(update.bugs)}</span></h3>
         % for build in update.builds:
         <tr class="media">
           <td>
-            <a href="https://koji.fedoraproject.org/koji/search?terms=${build.nvr}&type=build&match=glob" target="_blank">
+            <a href="${request.registry.settings.get('koji_build_web_url')}?terms=${build.nvr}&type=build&match=glob" target="_blank">
```

Comment:
We need to do a ```urljoin``` here, so that we are forgiving if the admin does or does not set ```koji_web_url``` with a trailing slash. We will also need to include the ```/koji/search``` here since I suggested removing that from the setting above.